### PR TITLE
Translate block labels

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by core23** [comment]
@rande @Soullivaneuh @sagikazarmark Can you help me with the translation?

File: Resources/translations/SonataDashboardBundle.de.xliff [hunk 1]
```diff
@@ -197,7 +197,23 @@
...
+                <source>form.label_code</source>
+                <target>Code</target>
+            </trans-unit>
+            <trans-unit id="50" resname="form.label_layout">
+                <source>form.label_layout</source>
+                <target>Layout</target>
+            </trans-unit>
+            <trans-unit id="51" resname="form.label_class">
+                <source>form.label_class</source>
+                <target>Class</target>
```
Comment (OskarStark) [inline Resources/translations/SonataDashboardBundle.de.xliff]: Klasse?

Comment (core23) [inline Resources/translations/SonataDashboardBundle.de.xliff]: The ``class`` label is for *css classes*. Maybe **CSS Class** or **CSS Klasse**?

File: Resources/translations/SonataDashboardBundle.fr.xliff [hunk 2]
```diff
@@ -198,6 +198,22 @@
                 <source>compose.status_error</source>
                 <target>compose.status_error</target>
             </trans-unit>
+            <trans-unit id="49" resname="form.label_code">
+                <source>form.label_code</source>
+                <target>form.label_code</target>
```
Comment (qsomazzi) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: Code

File: Resources/translations/SonataDashboardBundle.fr.xliff [hunk 3]
```diff
@@ -198,6 +198,22 @@
...
+            </trans-unit>
+            <trans-unit id="50" resname="form.label_layout">
+                <source>form.label_layout</source>
+                <target>form.label_layout</target>
+            </trans-unit>
+            <trans-unit id="51" resname="form.label_class">
+                <source>form.label_class</source>
+                <target>form.label_class</target>
```
Comment (qsomazzi) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: Class

File: Resources/translations/SonataDashboardBundle.fr.xliff [hunk 4]
```diff
@@ -198,6 +198,22 @@
...
+            </trans-unit>
+            <trans-unit id="52" resname="form.label_template">
+                <source>form.label_template</source>
+                <target>form.label_template</target>
```
Comment (qsomazzi) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: Modèle de dashboard

File: Resources/translations/SonataDashboardBundle.fr.xliff [hunk 5]
```diff
@@ -198,6 +198,22 @@
                 <source>compose.status_error</source>
                 <target>compose.status_error</target>
             </trans-unit>
+            <trans-unit id="49" resname="form.label_code">
+                <source>form.label_code</source>
+                <target>form.label_code</target>
+            </trans-unit>
+            <trans-unit id="50" resname="form.label_layout">
+                <source>form.label_layout</source>
+                <target>form.label_layout</target>
```
Comment (qsomazzi) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: Structure ? (not sure for this one)

File: Resources/translations/SonataDashboardBundle.de.xliff (hunk 1 above)
Comment (OskarStark) [inline Resources/translations/SonataDashboardBundle.de.xliff]: ``CSS Klasse`` :+1:

File: Resources/translations/SonataDashboardBundle.fr.xliff (hunk 5 above)
Comment (OskarStark) [inline Resources/translations/SonataDashboardBundle.fr.xliff]: ping @Soullivaneuh @rande

File: Resources/translations/SonataDashboardBundle.en.xliff [hunk 6]
```diff
@@ -197,7 +197,23 @@
...
+                <source>form.label_code</source>
+                <target>Code</target>
+            </trans-unit>
+            <trans-unit id="50" resname="form.label_layout">
+                <source>form.label_layout</source>
+                <target>Layout</target>
+            </trans-unit>
+            <trans-unit id="51" resname="form.label_class">
+                <source>form.label_class</source>
+                <target>Class</target>
```
Comment (OskarStark) [inline Resources/translations/SonataDashboardBundle.en.xliff]: ``CSS class``

**Comment by rande** [comment]
form.label_code => code
form.label_layout => affichage
form.label_class => Classe CSS
form.label_template => Modèle de page

**Comment by core23** [comment]
Thank you @rande

**Comment by qsomazzi** [comment]
:+1:

**Comment by qsomazzi** [comment]
Thanks @core23 !

**Pull request merged**